oc-chef-pedant/lib/pedant.rb

```diff
@@ -85,8 +85,8 @@ def self.configure_logging
   end
 
   def self.create_platform
-    superuser_key = ENV['CHEF_SECRET_CHEF-SERVER.SUPERUSER_KEY'] || ENV['SUPERUSER_KEY']
-    webui_key = ENV['CHEF_SECRET_CHEF-SERVER.WEBUI_KEY'] || ENV['WEBUI_KEY']
+    superuser_key = ENV["#{ChefConfig::Dist::SHORT.upcase}_SECRET_#{ChefConfig::Dist::SERVER.upcase}.SUPERUSER_KEY"] || ENV['SUPERUSER_KEY']
```

Comment:
```
chef-server-ctl test
Configuring logging...
Creating platform...
bundler: failed to load command: ./bin/oc-chef-pedant (./bin/oc-chef-pedant)
NameError: uninitialized constant Pedant::ChefConfig
```

Reply:
`ChefConfig::Dist::SERVER` does not exist i replaced it with 
`ChefConfig::Dist::SHORT.upcase + '-SERVER'` and added `require 'chef-config/dist'
` to `oc-chef-pedant` to get it going.